gradle.properties

```diff
@@ -10,16 +10,16 @@ lombokVersion=1.18.10
 pageHelperVersion=1.2.4
 mockitoVersion=2.8.47
 guavaVersion=21.0
-pulsarVersion=2.5.2
+pulsarVersion=2.7.0
 swagger2Version=2.9.2
 swaggeruiVersion=2.9.2
 apiMockitoVersion=1.7.1
 mockitoJunit4Version=1.7.1
 gsonVersion=2.8.2
 postgresqlVersion=42.2.5
-herddbVersion=0.19.0
+herddbVersion=0.22.0
 brokerVersion=2.4.1
```

Comment:
can you please check this variable ?
it looks like it is another variable to refer to Pulsar.
I suggest to remove this variable and replace the reference to it with a reference to pulsarVersion

Reply:
You're right. I removed the variable brokerVersion and replaced it with pulsarVersion.